modules/ocf_docker/manifests/init.pp

```diff
@@ -2,6 +2,7 @@
   include ocf::firewall::allow_web
   include ocf::packages::docker
   require ocf::ssl::default
+  include ocf_docker::deckschrubber
```

Comment:
I'd probably just include this above the ssl thing to make this look more uniform, but it doesn't really make a difference, I just think it would look a little nicer :P